### DISCUSSION TITLE
Quarto render correctly but 404 on Github Pages

### DISCUSSION THREAD

**Discussion by alanmaulanaa** [quarto-dev/quarto-cli#9828] [Q&A] [answered]
### Description

I know this might be basics for some people but my Quarto Preview is fine, the only things is when I try to preview it in Github Pages it's always show 404 Not Found.

This is how my _quarto.yml looks like
````qmd
project:
  type: website
  output-dir: .

website:
  title: "Alan.dev"
  navbar:
    background: primary
    search: true
    left:
      - text: "Home"
        href: index.qmd
      - text: "Talks"
        href: talks.qmd
      - text: "About"
        href: about.qmd 

format:
  html:
    theme: cosmo
    css: styles.css
    toc: true
````
![quarto](https://github.com/quarto-dev/quarto-cli/assets/123195469/c6a3c254-c2c1-4807-856f-0c5d0bff0668)

I also already try using
````qmd
project:
  output-dir: docs
````
and change the folder from `/root` to `/docs` in the Branch, but in this case, there's wrong with something like Jekyll theme and cause the deployment fail.

`Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
                    No such file or directory @ dir_chdir - /github/workspace/docs
/usr/local/bundle/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:86:in `chdir': No such file or directory @ dir_chdir - /github/workspace/docs (Errno::ENOENT)
	from /usr/local/bundle/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:86:in `sass_load_paths'
	from /usr/local/bundle/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:1[14](https://github.com/alanmaulanaa/myportosite/actions/runs/9322344167/job/25663228102#step:4:15):in `sass_configs'
	from /usr/local/bundle/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:119:in `convert'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/renderer.rb:101:in `block in convert'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/renderer.rb:99:in `each'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/renderer.rb:99:in `reduce'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/renderer.rb:99:in `convert'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/renderer.rb:83:in `render_document'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/renderer.rb:62:in `run'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/site.rb:479:in `render_regenerated'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/site.rb:472:in `block in render_pages'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/site.rb:471:in `each'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/site.rb:471:in `render_pages'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/site.rb:192:in `render'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/site.rb:71:in `process'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/command.rb:28:in `process_site'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/commands/build.rb:65:in `build'
	from /usr/local/bundle/gems/jekyll-3.9.5/lib/jekyll/commands/build.rb:36:in `process'
	from /usr/local/bundle/gems/github-pages-231/bin/github-pages:70:in `block (3 levels) in <top (required)>'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
	from /usr/local/bundle/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
	from /usr/local/bundle/gems/github-pages-231/bin/github-pages:6:in `<top (required)>'
	from /usr/local/bundle/bin/github-pages:23:in `load'
	from /usr/local/bundle/bin/github-pages:23:in `<main>'
Error:  Logging at level: debug GitHub Pages: github-pages v231 GitHub Pages: jekyll v3.9.5 Theme: jekyll-theme-primer Theme source: /usr/local/bundle/gems/jekyll-theme-primer-0.6.0 Requiring: jekyll-github-metadata Requiring: jekyll-seo-tag Requiring: jekyll-coffeescript Requiring: jekyll-commonmark-ghpages Requiring: jekyll-gist Requiring: jekyll-github-metadata Requiring: jekyll-paginate Requiring: jekyll-relative-links Requiring: jekyll-optional-front-matter Requiring: jekyll-readme-index Requiring: jekyll-default-layout Requiring: jekyll-titles-from-headings GitHub Metadata: Initializing... Source: /github/workspace/./docs Destination: /github/workspace/./docs/_site Incremental build: disabled. Enable with --incremental Generating... Generating: JekyllOptionalFrontMatter::Generator finished in 7.944e-06 seconds. Generating: JekyllReadmeIndex::Generator finished in 4.128e-06 seconds. Generating: Jekyll::Paginate::Pagination finished in 2.485e-06 seconds. Generating: JekyllRelativeLinks::Generator finished in 2.4386e-05 seconds. Generating: JekyllDefaultLayout::Generator finished in 9.7[18](https://github.com/alanmaulanaa/myportosite/actions/runs/9322344167/job/25663228102#step:4:19)e-06 seconds. Generating: JekyllTitlesFromHeadings::Generator finished in 5.761e-06 seconds. Rendering: assets/css/style.scss Pre-Render Hooks: assets/css/style.scss Rendering Markup: assets/css/style.scss github-pages 231 | Error: No such file or directory @ dir_chdir - /github/workspace/docs`

This is my repo : [Repo](https://github.com/alanmaulanaa/myportosite)

**Comment by cscheid** [comment, marked as answer]
> Conversion error: Jekyll::Converters::Scss encountered

You shouldn't be using Jekyll. See https://quarto.org/docs/publishing/github-pages.html#render-to-docs
